packages/plugins/GoPlusSecurity/src/SNSAdaptor/index.tsx

```diff
@@ -12,7 +12,7 @@ const sns: Plugin.SNSAdaptor.Definition = {
     ApplicationEntries: [
         (() => {
             const icon = <SecurityCheckerIcon />
-            const name = { i18nKey: '__plugin_name', fallback: 'Check Security' }
```

Comment:
Why you're removing those keys? Please revert it.

Reply:
done